src/permission/PermissionStage.ts

```diff
@@ -0,0 +1,23 @@
+import { Stack, Stage } from 'aws-cdk-lib';
+import { StringParameter } from 'aws-cdk-lib/aws-ssm';
+import { Statics } from '../statics';
+
+export class PermissionStage extends Stage {
```

Comment:
Is het nodig dat dit een hele eigen stage is? Betekent ook dat ie niet parallel aan andere stacks gedeployeerd kan worden.